catalog.html

```diff
@@ -170,22 +177,22 @@ <h3>Amara Resort & Spa</h3>
               <span>От 4000 ₽</span>
             </p>
             <p>
-              <a class="button" href="#">Подробнее</a>
+              <a class="button more-details-button" href="#">Подробнее</a>
               <a class="button add-to-fav-button" href="#">В избранное</a>
             </p>
           </div>
           <div class="item-rating">
-              <img src="images/star.svg" width="18" alt="Классификация места проживания: 4 звезды">
-              <img src="images/star.svg" width="18">
-              <img src="images/star.svg" width="18">
-              <img src="images/star.svg" width="18">
-              <b>Рейтинг: 8,5</b>
+              <img src="images/star.svg" width="18" height="17" alt="Классификация места проживания: 4 звезды">
```

Comment:
Тут наверно лучше так
```suggestion
              <img src="images/star.svg" width="18" height="17" alt="Звезда">
```

Если картинки не будут загружаться - alt покажет 4 надписи "Звезда" и будет понятно